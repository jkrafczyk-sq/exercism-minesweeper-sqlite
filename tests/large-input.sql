INSERT INTO input(line) VALUES 
    (' *  * '),
    ('  *   '),
    ('    * '),
    ('   * *'),
    (' *  * '),
    ('      ');
