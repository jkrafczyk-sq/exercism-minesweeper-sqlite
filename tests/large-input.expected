1*22*1
12*322
 123*2
112*4*
1*22*2
111111
